images/capi/ansible/roles/setup/tasks/debian.yml

```
@@ -16,6 +16,7 @@
   template:
     src: etc/apt/sources.list.j2
     dest: /etc/apt/sources.list
+    mode: 0600
```

Judging by the default mode of /etc/apt/sources.list on a new system, this should probably be:
```suggestion
    mode: 0644
```